tests/unit/local/test_lmdb_access.py

```diff
@@ -22,11 +23,19 @@
 
 
 def shard_and_get_keys(
-    lmdb_reader: yogadl.LmdbAccess, shard_index: int, world_size: int, sequential: bool
+    lmdb_reader: yogadl.LmdbAccess,
+    shard_index: int,
+    world_size: int,
+    sequential: bool,
+    drop_remainder: bool,
```

Comment:
let's be consistent on this name.  Either we always do `drop_shard_remainder` or we always shorten it to `drop_remainder` when we have a function with `shard` in its name (like LMDBDataRef._shard_keys()`)

Either option is fine with me.  I have a slight preference for using the exact same name everywhere, since it always has the exact same meaning.

Reply:
Done, also made `world_size` -> `num_shards` everywhere.